app/src/main/java/org/oppia/android/app/devoptions/markchapterscompleted/MarkChaptersCompletedFragmentPresenter.kt

```diff
@@ -23,7 +23,8 @@ class MarkChaptersCompletedFragmentPresenter @Inject constructor(
   private val activity: AppCompatActivity,
   private val fragment: Fragment,
   private val viewModelProvider: ViewModelProvider<MarkChaptersCompletedViewModel>,
-  private val modifyLessonProgressController: ModifyLessonProgressController
+  private val modifyLessonProgressController: ModifyLessonProgressController,
+  private val adapterFactory: BindableAdapter.MultiTypeBuilder.Factory
```

Comment:
At some places we are using `adapterFactory` and at other places we are naming it as `multiTypeBuilderFactory`.

I suggest keeping it consistent by using `multiTypeBuilderFactory` everywhere.

Reply:
This is understood, Updated.